fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/RootRdfContext.java

```diff
@@ -129,7 +132,25 @@ private void concatRepositoryTriples() {
                     createTypedLiteral(counters.get(PREFIX + FIXITY_REPAIRED_COUNTER).getCount()).asNode()));
         }
 
+        //add project build description triples here
+        final BuildPropertiesLoader buildLoader = new BuildPropertiesLoader();
+
+        final com.hp.hpl.jena.graph.Node releaseSubject = createURI(REPOSITORY_NAMESPACE + "release");
+            b.add(create(subject(), DOAP_RELEASE.asNode(), releaseSubject));
+            b.add(create(releaseSubject, type.asNode(), createURI(DOAP_NAMESPACE + "Version")));
```

Comment:
I'm not sure those triples are actually correct.I think you are conflating the repository software and this instance thereof. This repository instance wasn't created on `buildLoader.getBuildDate()`, the _software_ was. But triples with the repository root resource as their subject describe this repository _instance_, not the software.